components/rich-text.js

```diff
@@ -1,6 +1,6 @@
 import ReactMarkdown from 'react-markdown';
 
-export const RichText = ({ children, small = false }) => {
+export const RichText = ({ children, small = false, narrow = false }) => {
```

Comment:
I think it's better not to have different line heights so we don't need `narrow` and I feel like we don't need small also. if we need to override the style we can easily use `.someclass :global(.rich-text p) {margin: 0}`